ydb/docs/en/core/concepts/glossary.md

```diff
@@ -0,0 +1,515 @@
+
+The **transaction proxy** or `TX_PROXY` is a service that orchestrates the execution of many [distributed transactions](#distributed-transactions): sequential phases, phase execution, planning, and aggregation of results. In the case of direct orchestration by other actors (for example, KQP data transactions), it is used for caching and allocation of unique [TxIDs](#txid).
+
+#### Transaction flags {#txflags}
+
+**Transaction flags** or **TxFlags** is a bitmask of flags that modify the execution of a transaction in some way.
+
+#### Transaction ID {#txid}
+
+**Transaction ID** or **TxID** is a unique identifier assigned to each transaction when it is accepted by {{ ydb-short-name }}.
+
+#### Transaction order ID {#transaction-order-id}
+
+A **transaction order ID** is a unique identifier assigned to each transaction during planning.
+
+#### PlanStep {#planstep}
+
+**PlanStep** or **step** is the logical time for which a set of transactions is planned to be executed.
+
+#### Mediator time {#mediator-time}
+
+**Mediator time** is the logical time before which (inclusive) the shard participant must know the entire execution plan. It is used to advance the time in the absence of transactions on a particular shard to determine whether it can read from a snapshot.
+
+#### MiniKQL {#minikql}
+
+**MiniKQL** is a language that allows the expression of a single [deterministic transaction](#deterministic-transactions) in the system. It is a functional, strongly typed language. Conceptually, the language describes a graph of reading from the database, performing calculations on the read data, and writing the results to the database and/or to a special document representing the query result (shown to the user). The MiniKQL transaction must explicitly set its read set (readable data) and assume a deterministic selection of execution branches (for example, there is no random).
+  
+MiniKQL is a low-level language. The system's end users only see queries in the [YQL](#yql) language, which relies on MiniKQL in its implementation.
+
+#### KQP {#kqp}
+
+**KQP** is a {{ ydb-short-name }} component responsible for the orchestration of user query execution and generating the final response.
+
+### Global schema {#global-schema}
+
+**Global Schema**, **Global Scheme**, or **Database Schema** is a schema of all the data stored in a [database](#database). It consists of [tables](#table) and other entities, such as [topics](#topic). The metadata about these entities is called a global schema. The term is used in contrast to **Local Schema**, which refers to the data schema inside a [tablet](#tablet). {{ ydb-short-name }} users never see the local schema and only work with the global schema.
+
+### KiKiMR {#kikimr}
+
+**KiKiMR** is the legacy name of {{ ydb-short-name }} that was used before it became an [open-source product](https://github.com/ydb-platform/ydb). It can still be occasionally found in the source code, old articles and videos, etc.
```

Comment:
```suggestion
**KiKiMR** is the legacy name of {{ ydb-short-name }} that was used long before it became an [open-source product](https://github.com/ydb-platform/ydb). It can still be occasionally found in the source code, old articles and videos, etc.
```